src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/CommandProcessor.java

```diff
@@ -1641,6 +1641,41 @@ public void epilogue() {
                 }
             }
         },
+        new Command("testdebuginfodecode", "testdebuginfodecode", false) {
```

Comment:
This doesn't belong in clhsdb. You can do all of this directly in the test if you launch it properly. 
See for example `TestObjectAlignment.java`.

Reply:
Yes that's much better.  I've updated the test and renamed it.  Please rereview.